pom.xml

```diff
@@ -168,39 +168,39 @@
                 <version>${shrinkwrap.descriptors.version}</version>
             </dependency>
             <dependency>
-                <groupId>javax.enterprise</groupId>
-                <artifactId>cdi-api</artifactId>
+                <groupId>jakarta.enterprise</groupId>
+                <artifactId>jakarta.enterprise.cdi-api</artifactId>
```

Comment:
@starksm64 if you can do anything about that, I would recommend we keep the artifactId as `cdi-api` to ease a bit the transition. At any rate, jakarta.enterprise seems to be used by noone else.

Reply:
These artifacts have already been pushed to Maven Central and it's the names that have been provided to us.

Reply:
@starksm64 @emmanuelbernard 

I repeat what I wrote in the bean validation-api review so that Scott has the information too:

We decided to go this way for the ones we already published. All the Jakarta artifacts are `jakarta.something`.

See https://search.maven.org/search?q=g:jakarta.validation
and https://search.maven.org/search?q=g:jakarta.*

Reply:
I'm making my case here https://bugs.eclipse.org/bugs/show_bug.cgi?id=548104#c3

Reply:
Yeah, frankly, being inconsistent with the other specs is really a bad idea IMHO.

That's the good thing with the Jakarta specs right now: all the groupIds and artifacts are consistent and that is important.

Moreover we already released Jakarta artifacts with this naming so changing it now will create confusion.

Reply:
Oh, and not changing it now means being inconsistent forever.

It's better to change it when people have to change the groupId too.